Automattic/jetpack
# Woo Analytics: auto-label package changes with the feature label

## Proposed changes:

The WooCommerce Analytics feature now lives inside a package as well as a Jetpack module. Let's auto-label changes to both with the "[Feature] WooCommerce Analytics" label.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* This cannot be tested in this repo before it's merged.
* In a fork, add the changes from this PR to `trunk`.
* Make changes to the files in `projects/packages/woocommerce-analytics`
* Open a PR
    * The "[Feature] WooCommerce Analytics" label should be automatically added.
